core/controller/src/main/scala/whisk/core/controller/ApiUtils.scala

```diff
@@ -126,9 +124,7 @@ trait ReadOps extends Directives {
   /**
    * Gets an entity of type A from datastore. Terminates HTTP request.
```

Comment:
the method now seems to wait for entity response from future instead of doing the actual get/retrieval...
> Gets an entity